src/profile-logic/symbolication.js

```diff
@@ -281,6 +281,7 @@ export async function symbolicateProfile(
         throw error;
       }
       // We could not find symbols for this library.
+      console.log(error);
```

Comment:
Would `console.error` be better ? (not a suggestion, I don't really know)

Reply:
I tried `console.warn` first, but then I got these messages printed during `yarn test` runs, with very long stacks, and it was annoying. I don't see it printed during `yarn test` if I use `console.log`.

Reply:
This will probably be that we've mocked console log in the tests already, but not console.warn.

Reply:
Tests: I think this line should have an assertion similar to `expect(console.log).toHaveBeenCalledTimes(1);` in the tests.